web/src/components/vis/VisTile.vue

```diff
@@ -11,28 +11,61 @@ export default {
     },
   },
 
+  data() {
+    return {
+      scale: 1,
+    };
+  },
+
   computed: {
-    hasControls() {
-      return !!this.$slots.controls;
+    smallEnough() {
+      return this.scale <= 0.5;
+    },
+    largeEnough() {
+      return this.scale >= 4;
```

Comment:
Perhaps refactor `minScale` and `maxScale` to constants at the top of the script.

Reply:
This discussion has moved to the thread below.